core/src/main/antlr4/io/delta/sql/parser/DeltaSqlBase.g4

```diff
@@ -81,14 +81,16 @@ statement
     | CONVERT TO DELTA table=qualifiedName
         (PARTITIONED BY '(' colTypeList ')')?                           #convert
     | RESTORE TABLE? table=qualifiedName TO?
-            clause=temporalClause                                       #restore
+        clause=temporalClause                                           #restore
     | ALTER TABLE table=qualifiedName ADD CONSTRAINT name=identifier
-      constraint                                                        #addTableConstraint
+        constraint                                                      #addTableConstraint
     | ALTER TABLE table=qualifiedName
         DROP CONSTRAINT (IF EXISTS)? name=identifier                    #dropTableConstraint
     | OPTIMIZE (path=STRING | table=qualifiedName)
         (WHERE partitionPredicate = exprToken)?
         (zorderSpec)?                                                   #optimizeTable
+    | SHOW COLUMNS (IN | FROM) (path=STRING | tableName=qualifiedName
```

Comment:
we probably don't want to support `show columns '/foo/bar'` given spark doesn't support it either.

Reply:
This is no longer supported.